src/retrace-server-reposync

```diff
@@ -163,17 +163,17 @@ def clean_rawhide_repo(release):
 
         packages[pkgdata["name"]][ver] = [f.name]
 
-    for package in packages:
-        pkgcnt = len(packages[package])
+    for pkg in packages:
+        pkgcnt = len(packages[pkg])
         if pkgcnt > CONFIG["KeepRawhideLatest"]:
-            vers = sorted(packages[package].keys(), key=cmp_to_key(vercmp))
-            i = 0
+            vers = sorted(packages[pkg].keys(), key=cmp_to_key(vercmp))
+            n = 0
             for ver in vers:
```

Comment:
This could perhaps be better rewritten using `enumerate()`.

Reply:
Like this?

Reply:
The other way around: `for n, ver in enumerate(vers):` I'd keep the definition of `vers` outside for clarity.